pygtracker/pygtracker.py

```diff
@@ -23,6 +24,23 @@ def __init__(self):
     # End: Summarize the grades by courses
 
     # Start: Summarise the grades by students
+    def student_ranker(course_id="all", n=10, descending=True):
```

Comment:
```suggestion
    def rank_students(course_id="all", n=10, descending=True):
```

normally we name the function as a verb: